src/cpp/src/visual_language/pipeline.cpp

```diff
@@ -348,6 +348,17 @@ class VLMPipeline::VLMPipelineImpl : public VLMPipelineBase{
     }
 };
 
+// TODO: remove it when CVS-167316 is fixed
+bool requires_sdpa(const std::filesystem::path& models_dir) {
+    auto vlm_config = utils::from_config_json_if_exists<VLMConfig>(models_dir, "config.json");
+    if (vlm_config.model_type == VLMModelType::QWEN2_VL) {
+        return true;
+    } else if (vlm_config.model_type == VLMModelType::QWEN2_5_VL) {
+        return true;
+    }
+    return false;
```

Comment:
Just a small nit/refactor
```suggestion
    return vlm_config.model_type == VLMModelType::QWEN2_VL || 
           vlm_config.model_type == VLMModelType::QWEN2_5_VL;
```
or
```suggestion
    static const std::unordered_set<VLMModelType> sdpa_required_models = {
        VLMModelType::QWEN2_VL,
        VLMModelType::QWEN2_5_VL
    };
    return sdpa_required_models.contains(vlm_config.model_type);
```